### PR TITLE
#174797453 Implement multiple languages support

### PR DESCRIPTION
**Pull request by dushimeemma**
#### What does this PR do?
- Add the functionality to have multiple languages
#### Description of Task to be completed?
Have the following steps
- install i18n dependency `npm i i18n --save`
- configure the app to support internationalization using `i18n` dependency
- configure the app to support three languages `english` ,`kinyarwanda` and `french`
#### How should this be manually tested?
- clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend/`
- enter into the directory `cd script-heroes-bn-backend`
- run the app `npm run dev`
- using `postman` send a get request to `http://localhost:3000`
- set `headers` `Accept-Language` with `en` or `kin` or `fr`
#### Any background context you want to provide?
- `en` stands for `english`
- `kin` stands for `kinyarwanda`
- `fr` stands for `french`
#### What are the relevant pivotal tracker stories?
[#174797453](https://www.pivotaltracker.com/n/projects/2465784/stories/174797453)
#### Screenshots (if appropriate)
Screenshot from 2020-09-23 10-07-51
Screenshot from 2020-09-23 10-06-56
Screenshot from 2020-09-23 10-08-22

### CODE REVIEW

File: src/utils/internationalization/locales/en.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"Welcome to barefoot nomad": "Welcome to barefoot nomad"
```
Comment (elvisiraguha) [inline src/utils/internationalization/locales/en.json]: Fix the new line issue at the end of this file.
Also, there is an indent difference on this line between this file and [fr.json](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/ft-internationalization-174797453/src/utils/internationalization/locales/fr.json) or [kin.json](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/ft-internationalization-174797453/src/utils/internationalization/locales/kin.json)

Comment (dushimeemma) [reply src/utils/internationalization/locales/en.json]: it is solved now, i think?

**Review by Habinezajanvier — APPROVED** [review]
Ok

File: src/utils/internationalization/locales/en.json (hunk 1 above)
Comment (elvisiraguha) [reply src/utils/internationalization/locales/en.json]: :+1:

**Review by elvisiraguha — APPROVED** [review]

**Comment by Adadie** [comment]
> #### What does this PR do?
> * Add the functionality to have multiple languages
> 
> #### Description of Task to be completed?
> Have the following steps
> 
> * install i18n dependency `npm i i18n --save`
> * configure the app to support internationalization using `i18n` dependency
> * configure the app to support three languages `english` ,`kinyarwanda` and `french`
> 
> #### How should this be manually tested?
> * clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend/`  
> * enter into the directory `cd script-heroes-bn-backend` 

> **"Change this line to checkout not enter into directory"**

> * run the app `npm run dev`
> * using `postman` send a get request to `http://localhost:3000`
> * set `headers` `Accept-Language` with `en` or `kin` or `fr`
> 
> #### Any background context you want to provide?
> * `en` stands for `english`
> * `kin` stands for `kinyarwanda`
> * `fr` stands for `french`
> 
> #### What are the relevant pivotal tracker stories?
> [#174797453](https://www.pivotaltracker.com/n/projects/2465784/stories/174797453)
> 
> #### Screenshots (if appropriate)
> Screenshot from 2020-09-23 10-07-51
> Screenshot from 2020-09-23 10-06-56
> Screenshot from 2020-09-23 10-08-22

**Comment by Junior-Adadie** [comment]
> #### What does this PR do?
> * Add the functionality to have multiple languages
> 
> #### Description of Task to be completed?
> Have the following steps
> 
> * install i18n dependency `npm i i18n --save`
> * configure the app to support internationalization using `i18n` dependency
> * configure the app to support three languages `english` ,`kinyarwanda` and `french`
> 
> #### How should this be manually tested?
> * clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend/`
> * enter into the directory `cd script-heroes-bn-backend`
> * run the app `npm run dev`
> * using `postman` send a get request to `http://localhost:3000`
> * set `headers` `Accept-Language` with `en` or `kin` or `fr`
> 
> #### Any background context you want to provide?
> * `en` stands for `english`
> * `kin` stands for `kinyarwanda`
> * `fr` stands for `french`
> 
> #### What are the relevant pivotal tracker stories?
> [#174797453](https://www.pivotaltracker.com/n/projects/2465784/stories/174797453)
> 
> #### Screenshots (if appropriate)
> Screenshot from 2020-09-23 10-07-51
> Screenshot from 2020-09-23 10-06-56
> Screenshot from 2020-09-23 10-08-22

**Review by valens10 — APPROVED** [review]

**Pull request merged**